packages/message-box/doc/msgbox.vue

```diff
@@ -0,0 +1,50 @@
+<template>
```

Comment:
you can simply remove this folder `packages/message-box/doc/*`. We are now using the doc website to dev and preview.

Reply:
ok i already deleted